projects/coordinator/src/main/java/org/batfish/coordinator/WorkMgrService.java

```diff
@@ -555,6 +555,46 @@ public JSONArray getAnswer(
       }
    }
 
+   @POST
+   @Path(CoordConsts.SVC_RSC_GET_CONTAINER)
+   @Produces(MediaType.APPLICATION_JSON)
+   public JSONArray getContainer(
```

Comment:
Now the command used is 'exist-container', will output a string whether indicates the container does not exist, or the creation time of the container